Simplenote/Information/NoteInformationController.swift

```diff
@@ -141,12 +141,14 @@ private extension NoteInformationController {
             return []
         }
 
-        let referenceRows = references.map { (note) -> Row in
-            let date = DateFormatter.dateFormatter.string(from: note.modificationDate)
-            let value = "\(Localization.interlinkReferences(references.count)), \(Localization.lastModified(date))"
-
-            return .reference(interLink: note.plainInternalLink,
-                              title: note.titlePreview,
+        let referenceRows = references.map { (referenceNote) -> Row in
+            let date = DateFormatter.dateFormatter.string(from: referenceNote.modificationDate)
+            let value = Localization.interlinkReferences(Note.instancesOfReferenceToNoteWith(simperiumKey: note.simperiumKey, in: referenceNote.content)) +
```

Comment:
For readability purposes, can we move the `Note.instancesOfReferenceToNoteWith()` invocation up, and assign that to a constant?

Reply:
Yes we can.